# Fixes gentoo bug 541654
# https://bugs.gentoo.org/show_bug.cgi?id=541654
# Committed on behalf of Luca Barbato <lu_zero@gentoo.org>

--- modules/codec/avcodec/avcommon.h.orig	2015-02-28 15:19:33.449761287 +0000
+++ modules/codec/avcodec/avcommon.h	2015-02-28 15:20:10.469537179 +0000
@@ -81,6 +81,7 @@
 #ifdef HAVE_LIBAVUTIL_AVUTIL_H
 # include <libavutil/avutil.h>
 # include <libavutil/dict.h>
+# include <libavutil/mem.h>
 
 #define AV_OPTIONS_TEXT     "Advanced options."
 #define AV_OPTIONS_LONGTEXT "Advanced options, in the form {opt=val,opt2=val2} ."
